streams/runners/storm/runtime/src/test/java/com/hortonworks/streamline/streams/runtime/AvroStreamsSnapshotDeserializerTest.java

```diff
@@ -66,7 +68,10 @@ public void testAvroPayloadConversions() throws Exception {
 
             new Expectations() {
                 {
-                    mockSchemaRegistryClient.getSchemaMetadataInfo(anyLong);
+                    mockSchemaRegistryClient.getSchemaVersionInfo(withEqual(SCHEMA_ID_VERSION));
```

Comment:
Not that important really. Its just a little stricter than before. It just makes sure that its called with same SchemaIdVersion as the one used for serialization.